qt/applications/workbench/workbench/plotting/figuremanager.py

```diff
@@ -399,10 +399,11 @@ def fit_toggle(self):
         """Toggle fit browser and tool on/off"""
         if self.fit_browser.isVisible():
             self.fit_browser.hide()
-            self.toolbar._actions["toggle_superplot"].setEnabled(True)
+            self.toolbar._actions["toggle_fit"].setChecked(False)
         else:
+            if self.toolbar._actions["toggle_superplot"].isChecked():
+                self.superplot_toggle()
```

Comment:
This is me bike shedding, but toggle implies switching state on/off, should this be called `superplot_enable()` instead?
The current method name looks like we should call toggle every time a user toggles the toolbar button on / off currently, rather than just leaving it on

Reply:
I split the toggle function in two: `superplot_show` and `superplot_hide`; and called `superplot_hide` here.